vllm/v1/executor/multiproc_executor.py

```diff
@@ -593,11 +595,11 @@ class ResponseStatus(Enum):
         SUCCESS = auto()
         FAILURE = auto()
 
-    def worker_busy_loop(self):
+    def worker_busy_loop(self, cancel: Optional[threading.Event] = None):
         """Main busy loop for Multiprocessing Workers"""
         while True:
-            method, args, kwargs, output_rank = self.rpc_broadcast_mq.dequeue()
-
+            method, args, kwargs, output_rank = self.rpc_broadcast_mq.dequeue(
+                cancel=cancel)
```

Comment:
@njhill How about this?

Reply:
I think only other thing is that in `worker_main` we don't want to log an error in this case because it's a normal shutdown. Right now that will happen when `RuntimeError("cancelled")` is caught